packages/client/src/views/OfficeHome/inProgress/InProgress.tsx

```diff
@@ -189,37 +194,36 @@ export class InProgressComponent extends React.Component<
 
       let name
       let eventDate = ''
-      if (reg.registration && reg.type === 'Birth') {
-        const birthReg = reg as GQLBirthEventSearchSet
-        const names = birthReg && (birthReg.childName as GQLHumanName[])
-        const namesMap = createNamesMap(names)
-        name = namesMap[locale] || namesMap[LANG_EN]
-        const date = (reg as GQLBirthEventSearchSet).dateOfBirth
-        eventDate = date && date
-      } else if (reg.registration && reg.type === 'Death') {
-        const deathReg = reg as GQLDeathEventSearchSet
-        const names = deathReg && (deathReg.deceasedName as GQLHumanName[])
-        const namesMap = createNamesMap(names)
-        name = namesMap[locale] || namesMap[LANG_EN]
-        const date = (reg as GQLDeathEventSearchSet).dateOfDeath
-        eventDate = date && date
-      } else if (reg.registration && reg.type === 'Marriage') {
-        const marriageReg = reg as GQLMarriageEventSearchSet
-        const groomNames =
-          marriageReg && (marriageReg.groomName as GQLHumanName[])
-        const groomNamesMap = createNamesMap(groomNames)
-        const brideNames =
-          marriageReg && (marriageReg.brideName as GQLHumanName[])
-        const brideNamesMap = createNamesMap(brideNames)
-        const groomName = groomNamesMap[locale] || groomNamesMap[LANG_EN]
-        const brideName = brideNamesMap[locale] || brideNamesMap[LANG_EN]
-        name =
-          brideName && groomName
-            ? `${groomName} & ${brideName}`
-            : brideName || groomName || EMPTY_STRING
-        const date = (reg as GQLMarriageEventSearchSet).dateOfMarriage
-        eventDate = date && date
+
+      if (reg.registration) {
+        if (isBirthEvent(reg)) {
+          const names = reg.childName as GQLHumanName[]
```

Comment:
Do we not want to remove the `GQLHumanName` from here?

Reply:
The reason why I left this "`as GQLHumanName[]`" there was that the field `reg.childName` is possibly undefined and the createNamesMap function expect the argument type to be `createNamesMap = (names: HumanName[])`